### PR TITLE
Update dependencies list to preference `main` over `master`.

### PR DESCRIPTION
**Pull request by rfk**
Our periodic dependency-check task is failing in CI due to this exogenous change, this should fix it.

### CODE REVIEW

**Comment by rfk** [comment]
(Also picking up a fix for a brand new RUSTSEC advisory while I'm here)

File: megazords/full/android/dependency-licenses.xml [hunk 1]
```diff
@@ -50,19 +50,19 @@ the details of which are reproduced below.
   </license>
   <license>
     <name>Apache License 2.0: askama</name>
-    <url>https://github.com/djc/askama/blob/main/LICENSE-APACHE</url>
+    <url>https://github.com/djc/askama/blob/master/LICENSE-APACHE</url>
```
Comment (mhammond) [inline megazords/full/android/dependency-licenses.xml:53]: These changes look reversed? The 'main' branch exists in all these repos, but this patch is changing it to the old 'master'?

Comment (rfk) [reply megazords/full/android/dependency-licenses.xml:53]: Oh, hrm, TBH I didn't look very hard I just re-ran the script. Let me see if I can figure out what's happening here...

**Comment by rfk** [comment]
OK, what I think is happening here is that there is a (maybe newly added?) redirect from the `master` branch to the `main` branch on github. When our dependency script tries to resolve a license URL, it checks `master` *first* and then falls back to `main`. The presence of this redirect means that `master` succeeds and we choose that as the URL.

I'm going to change things to preference `main` over `master`, which should avoid this issue.

**Review by mhammond — APPROVED** [review]

**Pull request merged**